crates/core_arch/src/aarch64/neon/mod.rs

```diff
@@ -2318,362 +2318,362 @@ pub unsafe fn vcvtq_u32_f32(a: float32x4_t) -> uint32x4_t {
 #[inline]
 #[target_feature(enable = "neon")]
 #[cfg_attr(test, assert_instr(sli, n = 1))]
-#[rustc_args_required_const(2)]
-pub unsafe fn vsli_n_s8(a: int8x8_t, b: int8x8_t, n: i32) -> int8x8_t {
-    assert!(0 <= n && n <= 7, "must have 0 ≤ n ≤ 7, but n = {}", n);
-    vsli_n_s8_(a, b, n)
+#[rustc_legacy_const_generics(2)]
+pub unsafe fn vsli_n_s8<const N: i32>(a: int8x8_t, b: int8x8_t) -> int8x8_t {
+    assert!(0 <= N && N <= 7, "must have 0 ≤ N ≤ 7, but N = {}", N);
```

Comment:
Use `static_assert!` instead of `assert!`.

Reply:
Thanks for pointing. It has been corrected.